bucket/vapoursyntheditor.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "r19",
```

Comment:
The 'r' is not required, only 19 is sufficient. You will need to change the URLs and checkver too for this.